src/enums.ts

```diff
@@ -34,17 +34,28 @@ export const COORDINATE_LABELS_GROUP_NAME = "Coordinate_Labels_Group";
 export const COLORS = {
     RED: 0xff0000,
     GREEN: 0x00ff00,
+    ORANGE: 0xffa500,
+    WHITE: 0xffffff,
+    BLACK: 0x000000,
 };
 
+export enum MEASUREMENT_MODES_ENUM {
+    NONE = "none",
+    DISTANCE = "distance",
+    ANGLE = "angle",
+    COORDINATE = "coordinate",
+}
+
 export const MEASUREMENT_MODES = {
     NONE: "none",
-    DISTANCE: "isDistanceShown",
-    ANGLE: "areAnglesShown",
-    COORDINATE: "areCoordinatesShown",
+    DISTANCE: "distance",
```

Comment:
Both `MEASUREMENT_MODES_ENUM` and `MEASUREMENT_MODES` represent the same enum. Should we leave only `MEASUREMENT_MODES_ENUM`?